operations/mimir-mixin/docs/playbooks.md

```diff
@@ -213,6 +217,11 @@ How to **investigate**:
         - Check `Memcached Overview` dashboard
         - If memcached eviction rate is high, then you should scale up memcached replicas. Check the recommendations by `Mimir / Scaling` dashboard and make reasonable adjustments as necessary.
         - If memcached eviction rate is zero or very low, then it may be caused by "first time" queries
+      - Cache query timeouts
+        - Check store gateway logs and look for warnings about timed out Memcached queries
+        - If there are indeed a lot of timed out Memcached queries, consider whether the store gateway Memcached timeout setting (`blocks-storage.bucket-store.chunks-cache.memcached.timeout`) is sufficient
+    - Consider increasing total number of query shards (`query_sharding_total_shards`) for tenants submitting slow queries, to increase query parallelism
+    - Consider scaling up number of queriers if they're not auto-scaled; if auto-scaled, check auto-scaling parameters
```

Comment:
If query-frontend OOM, the queries that were currently running on it will fail. If you run a gateway in front of it (like in Grafana Cloud) then gateway should report 5xx.

Reply:
@pracucci can you please review my edit here?